build-tools/xaprepare/xaprepare/Steps/Step_ThirdPartyNotices.cs

```diff
@@ -60,6 +60,11 @@ public Step_ThirdPartyNotices ()
 #pragma warning disable CS1998
 		protected override async Task<bool> Execute (Context context)
 		{
+			if (!string.IsNullOrEmpty (context.MonoArchiveCustomUrl)) {
+				Log.StatusLine ("Using custom mono archive, skipping third party notices.");
```

Comment:
Why skip the ThirdPartyNotices?  A custom mono archive should still contain them, shouldn't it?

Reply:
It might be this was just something broken on mono/master:

    Step Xamarin.Android.Prepare.Step_ThirdPartyNotices failed:
    License file external\mono\sdks\out\android-tpn\external\llvm\LICENSE.TXT does not exist

It looks like they moved the submodule? So maybe it's in `external\llvm-project` now?

https://github.com/mono/mono/tree/a2f9fb77a3704049ffe5de32362029ece154ee4c/external

I am building a mono archive based off 2019-10, will give that a try.

Reply:
Yes, the file moved to `external/llvm-project/llvm/LICENSE.TXT` in `mono/master` and in `mono/2019-12`

@jonathanpeppers I think instead of disabling TPN, you could make https://github.com/xamarin/xamarin-android/blob/master/build-tools/xaprepare/xaprepare/ThirdPartyNotices/mono.cs#L189 conditional on file existence